src/data/info.json

```diff
@@ -62,12 +139,24 @@
   {
     "id": 13,
     "title": "Hubert Whitfeld Sculpture",
-    "text": "A bronze sculpture of Professor Whitfeld looks over Whitfeld Court towards Winthrop Hall. The sculpture was commissioned in 1964 to honour his contributions to the welfare and well–being of the University as first Vice – Chancellor 1913 -15 and 1925 – 27 and 1927 – 39 and Foundation Professor of Mining Engineering in 1913; by South Australian artist John Dowie as a gift of Convocation in 1965. To commemorate the Centenary of Armistice on 11th November 2018, a Pavement Memorial was dedicated in Whitfeld Court in front of the Sculpture. The Roll of Honour in Winthrop Hall commemorates WW1 members of the University Family."
+    "text": "A bronze sculpture of Professor Whitfeld looks over Whitfeld Court towards Winthrop Hall. The sculpture was commissioned in 1964 to honour his contributions to the welfare and well–being of the University as first Vice – Chancellor 1913 -15 and 1925 – 27 and 1927 – 39 and Foundation Professor of Mining Engineering in 1913; by South Australian artist John Dowie as a gift of Convocation in 1965. To commemorate the Centenary of Armistice on 11th November 2018, a Pavement Memorial was dedicated in Whitfeld Court in front of the Sculpture. The Roll of Honour in Winthrop Hall commemorates WW1 members of the University Family.",
+    "images": [
+      {
+          "id": "26901P",
+          "caption": "26901P - Bust of Professor H E Whitfeld at Whitfeld Court"
+      }
+    ]
   },
   {
     "id": 14,
     "title": "Whitfeld Court",
-    "text": "In 1927, Leslie Wilkinson revised the original campus plan and established the fundamentals of the modern campus. The plan reoriented the campus to the north – south alignment that dominates today. This spine established the three major open spaces:  The Court of Honour named by Alsop (Whitfeld Court), The Great Court, and James Oval. Dr W Somerville promoted the concept that the area of land bounded on the north by Stirling Highway, on the south by Winthrop Hall, on the east by Hackett Hall, and on the west by the Administration Building to be named “Whitfeld Court”, so named by Senate in September 1940."
+    "text": "In 1927, Leslie Wilkinson revised the original campus plan and established the fundamentals of the modern campus. The plan reoriented the campus to the north – south alignment that dominates today. This spine established the three major open spaces:  The Court of Honour named by Alsop (Whitfeld Court), The Great Court, and James Oval. Dr W Somerville promoted the concept that the area of land bounded on the north by Stirling Highway, on the south by Winthrop Hall, on the east by Hackett Hall, and on the west by the Administration Building to be named “Whitfeld Court”, so named by Senate in September 1940.",
+    "images": [
```

Comment:
I'm not sure about using this image here, We'll ask fran

Reply:
If that was the picture of Whitfeld statue bust, that was a placeholder. 

I have corrected it with the latest commit. Sorry about that.

Reply:
Ah good good